packages/bpk-component-split-input/README.md

```diff
@@ -0,0 +1,43 @@
+# bpk-component-split-input
+
+> Backpack split-input component.
+
+## Installation
+
+```sh
+npm install bpk-component-split-input --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkSplitInput from 'bpk-component-code';
+import { INPUT_TYPES } from 'bpk-component-input';
+import { action } from '@storybook/addon-actions';
+
+export default () => (
+  <BpkSplitInput
+    name="otpInput"
+    id="otpInput"
+    label="otp input"
+    onChange={(e) => console.log('On Input Change')}
+    onSubmit={(e) => console.log('On Submit')}
+  />
+);
+```
+
+## Props
+
+
+| Property         | PropType                   | Required            | Default Value            |
+| ---------------- | -------------------------- | ------------------- | ------------------------ |
+| id               | string                     | true                | -                        |
+| name             | string                     | true                | -                        |
+| type             | INPUT_TYPES (one of)       | false               | INPUT_TYPES.number       |
+| inputLength      | number                     | false               | 4                        |
+| placeholder      | string                     | false               | -                        |
```

Comment:
defaultValue should be null when a prop is not required

Reply:
In `BpkSplitInput` the type is given a number default value although it's not a required parameter so I think it should be reflected here, and I noticed similar examples in other components. However happy to change it if this is a Backpack Convention :)

Reply:
Ah, sorry I misunderstood :)